### PR TITLE
Add read receipt times to the hovertip of read markers

### PR DESCRIPTION
**Pull request by Kegsay**
Fixes vector-im/riot-web#2709. Surprisingly, this data was never passed down to
`ReadReceiptMarker`.

### CODE REVIEW

**Comment by Kegsay** [comment]
I don't understand why Travis is upset with me.

File: src/components/views/rooms/EventTile.js [hunk 1]
```diff
@@ -290,6 +290,18 @@ module.exports = WithMatrixClient(React.createClass({
 
         var left = 0;
 
+        var readReceiptData = Object.create(null);
+        var room = this.props.matrixClient.getRoom(this.props.mxEvent.getRoomId());
+        if (room) {
+            // [ {type/userId/data} ]
+            room.getReceiptsForEvent(this.props.mxEvent).forEach(function(r) {
```
Comment (richvdh) [inline src/components/views/rooms/EventTile.js]: I'm not terribly happy about this being here as well as `this.props.readReceipts`. It gives a very odd interface to this component, which is becoming less and less 'view'-like.

Can we push this up into `MessagePanel` and add the timestamp data to `this.props.readReceipts`?

**Comment by richvdh** [comment]
(I don't know what's up with travis either; will investigate)

File: src/components/views/rooms/EventTile.js (hunk 1 above)
Comment (Kegsay) [inline src/components/views/rooms/EventTile.js]: I'm confused. I have not changed the interface to this component.

I dislike how `this.props.readReceipts` aren't read receipts at all, and are in fact a list of `RoomMembers`, which is why I have to contort like this. Are you asking for me to modify the interface to this component to make `this.props.readReceipts` actually be read receipts with an additional `roomMember` field?

Comment (richvdh) [inline src/components/views/rooms/EventTile.js]: yes, something like that.

**Comment by Kegsay** [comment]
@richvdh PTAL

**Pull request merged**